.travis.yml

```diff
@@ -2,13 +2,13 @@
 sudo: false
 dist: xenial
 language: java
-jdk:
-  - openjdk8
-  - openjdk11
-install: true
```

Comment:
According to [Travis](https://docs.travis-ci.com/user/job-lifecycle/#skipping-the-installation-phase), the `install: true` needs to stay in if you don't have anything to do in the install phase. This is because the install phase isn't optional.

Reply:
It seemed to work OK without out, but I'll put it back.

Reply:
In my experience, if you leave it out the `script` phase gets replaced with the default `gradle assemble`.